build/build-darwin.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
```

Comment:
The build can probably be done with grunt/gulp, but there will probably be additional steps required to package the final result, such as creating an installer, deb or dmg, which would be different enough on each platform that I felt separate shell scripts were reasonable.

You could use it as a reference going forward, I guess. I intended to write a bat file for windows, eventually, but work has killed my free time, so I don't think I'll be able to contribute much for the foreseeable future.